homeassistant/components/lifx/select.py

```diff
@@ -59,6 +59,7 @@ class LIFXInfraredBrightnessSelectEntity(LIFXEntity, SelectEntity):
     """LIFX Nightvision infrared brightness configuration entity."""
 
     _attr_has_entity_name = True
+    _attr_should_poll = False
```

Comment:
`LIFXInfraredBrightnessSelectEntity` inherits from `LIFXEntity` which inherits from `CoordinatorEntity` which inherits from `BaseCoordinatorEntity` which already sets `should_poll` to `False` so this looks like it will never be seen

https://github.com/home-assistant/core/blob/2f4325246b08225d2974dd7ca08245a780f91292/homeassistant/helpers/update_coordinator.py#L385